test/client/fixtures/sandbox/event/event-simulator-test.js

```diff
@@ -622,3 +623,63 @@ test('wrong type of the blur event (GH-947)', function () {
     eventSimulator.blur(domElement);
     ok(raised);
 });
+
+module('Drag and Drop');
```

Comment:
There is only one test for 'Drag and Drop'. 
Also we don't use `module` for grouping test of specified type. We use `module` only for regression tests.
So, let's move test `drag and drop events` test before `regression` module and remove `module('Drag and Drop');`